pb/protos/profile.proto

```diff
@@ -56,4 +49,20 @@ message Profile {
         string large    = 4;
         string original = 5;
     }
+
+    message Stats {
+        uint32 followerCount  = 1;
+        uint32 followingCount = 2;
+        uint32 listingCount   = 3;
+        uint32 avgRating      = 4;
+        uint32 numRatings     = 5;
```

Comment:
We have `followerCount`, `followingCount` and `listingCount`. Why suddenly `numRatings` as opposed to `ratingsCounts`? On this note, can `numRatings` come before `avgRating` so that all the counts are neatly bundled?